pkg/core/logger/logger.go

```diff
@@ -119,7 +126,7 @@ func DefaultLoggingConfig() *Config {
 	cfg.Beat = agentName
 	cfg.Level = DefaultLogLevel
 	cfg.ToFiles = true
-	cfg.Files.Path = paths.Logs()
+	cfg.Files.Path = filepath.Join(paths.Home(), DefaultLogDirectory) // paths.Logs()
```

Comment:
Wasn't `paths.Logs()` already equivalent to  `<home>/logs` ? :thinking: 
Also, what happens if we want to override the log location to somewhere different?

Reply:
paths.Logs is topPath by default

Reply:
Thanks, waiting for the fix, was struggling with the same issue:
https://discuss.elastic.co/t/elastic-agent-ignores-logging-level-setting/332560